### PR TITLE
Remove unused file and fix social media icon css on information page

### PR DESCRIPTION
**Pull request by Ridhwana**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
1. Removed the _login_cta_comment, this file is not used anywhere. 
2. As per [this chat in PR #7263](https://github.com/thepracticaldev/dev.to/pull/7263/files#r408026355), the SVG wasn't responding to these changes:
```
<%= inline_svg_tag("#{social_media_type}.svg", aria: true, title: social_media_type.capitalize, class: "crayons-icon social-media-svg") %>
```
```
.social-links .social-media-svg {
  width: 35px;
  height: 35px;
}
```
which basically sets the width and height of the icons to 35px on the `/p/information` page, and keeps it at 24 on the homepage.

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/assets/images/facebook.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M15.402 21v-6.966h2.333l.349-2.708h-2.682V9.598c0-.784.218-1.319 1.342-1.319h1.434V5.857a19.188 19.188 0 00-2.09-.107c-2.067 0-3.482 1.262-3.482 3.58v1.996h-2.338v2.708h2.338V21H4a1 1 0 01-1-1V4a1 1 0 011-1h16a1 1 0 011 1v16a1 1 0 01-1 1h-4.598z" /></svg>
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M15.402 21v-6.966h2.333l.349-2.708h-2.682V9.598c0-.784.218-1.319 1.342-1.319h1.434V5.857a19.188 19.188 0 00-2.09-.107c-2.067 0-3.482 1.262-3.482 3.58v1.996h-2.338v2.708h2.338V21H4a1 1 0 01-1-1V4a1 1 0 011-1h16a1 1 0 011 1v16a1 1 0 01-1 1h-4.598z" /></svg>
```
Comment (Ridhwana) [inline app/assets/images/facebook.svg:1]: Without the viewbox, the css on the elements aren't resizing the width and height of the svg as per my PR description

**Comment by ludwiczakpawel** [comment]
I have a PR waiting for social links here: https://github.com/thepracticaldev/dev.to/pull/7212 which basically should address the issues here.. I mean, we're not resizing these icons :P

**Comment by Ridhwana** [comment]
> I have a PR waiting for social links here: #7212 which basically should address the issues here.. I mean, we're not resizing these icons :P
Ah okay - @ludwiczakpawel you may have to pull in the latest master into your branch because the social media stuff is a now a partial which renders both on the homepage and the /p/information page, and I noticed that your branch does not have those changes. It was essentially the p/information page which needed the bigger icons as per the old design, I think this may still be needed?

**Comment by ludwiczakpawel** [comment]
Oh, gotcha!

You're right about both things:
- I need to merge master into my branch.
- Icons indeed need viewBox property!

Sooooo.... LGTM :)

File: app/assets/images/facebook.svg (hunk 1 above)
Comment (rhymes) [reply app/assets/images/facebook.svg:1]: I wonder if we should add a comment there, as that if the icons are ever updated we'll probably won't notice `viewBox` disappearing

Comment (Ridhwana) [reply app/assets/images/facebook.svg:1]: @rhymes usually whenever I've downloaded an svg it has a viewbox attribute, its pretty standard. I'm not sure why these didn't have them. It seems like a bit of an anomaly.

Comment (ludwiczakpawel) [reply app/assets/images/facebook.svg:1]: @rhymes 
yea - i blame myself for that.. i'll go through all our SVG icons and add viewBox attribute but in a separate PR :)

**Pull request merged**